assets/templates/dining_lists/snippet_diningslot.html

```diff
@@ -16,12 +16,12 @@
          {% if slot.association and slot.association.image %}style="background-image: url({{ slot.association.image.url }});"{% endif %}></div>
     <div class="col-8">
         <div class="text-size-5">
-        {% if slot.claimed_by %}
-            {{ slot.claimed_by }}
-            {% if slot.get_purchaser != slot.claimed_by %} and {{ slot.purchaser }}{% endif %}
-        {% else %}
-            Slot is unclaimed!
-        {% endif %}</div>
+            {% if slot.main_contact %}
+                {{ slot.main_contact }}
+            {% else %}
+                {{ slot.owners.all|join:', ' }}
+            {% endif %}
```

Comment:
I have noticed that when we use multiple names in the slot-snippet, a lot of space will be used for the names, which becomes a visual issue (or heavy annoyance) as it pushes away a lot of the remainder of the div. Which is why I have a preference for having one name. What we could also try is only use first names (association and the combination of names would likely create little conflict on who the chefs are. In the full info page we can off course display the full names of all users.

Reply:
For Quadrivium we will have name conflicts as we have for instance 3 people named Marieke who all 3 might join dinner on a Thursday. So we can see if it becomes a problem, however if you want to change it to first names I'll do that, I get that it's still possible to find the full names on the detail page.

Reply:
The first name bases is just for the dining_slot_snippet when no main contact is set. Yes, there could be some disparsity if a Marieke is one of the (multiple) owners and no main contact is added but then you can get more info on the information page.
That said, we can also make main_contact a property like so:
```
Class DiningList(parents):

_main_contact = models.ForeignKey(settings.AUTH_USER_MODEL,....)

@property
def main_contact(self):
   if self._main_contact == None:
      if self.owners.count() == 1
         return self.owners.first()
   return self._main_contact
```
That case, in case no main contact  is set, but there is only one owner it is automatically assumed main contact until more owners are added.

Reply:
> The first name bases is just for the dining_slot_snippet when no main contact is set.

I get that, that's also in my comment. Question remains: do you want me to change it to first names only? I'll make sure that it displays the full name if there's only 1 owner (but not using the `main_contact` property as that changes the semantic meaning of the main contact)

Reply:
Yes, please :)